src/main/java/net/openhft/chronicle/core/Jvm.java

```diff
@@ -1587,7 +1587,7 @@ public static String getCpuClass() {
                 if (Files.isReadable(path)) {
                     model = Files.lines(path)
                             .filter(line -> line.startsWith("model name"))
-                            .map(line -> line.replaceAll(".*: ", ""))
```

Comment:
does this code have tests?

Reply:
No. We should break out this functionality into a separate method and create a test.

Reply:
Added a test